### PR TITLE
Product list

### PR DESCRIPTION
**Pull request by theVivek-kumar**
image
image
image
image
image
image

### CODE REVIEW

**Review by eldhoabraham94 — APPROVED** [review]
Seems good to go

File: my-cart/product-list/product.css [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+.products>h1 {
+    padding-left: 1rem;
+}
+
+.products>div>div {
+    border: none;
+    border-radius: 4px;
+}
+
+
```
Comment (Adarsh-Patel-Dev) [inline my-cart/product-list/product.css:80]: remove extra spaces

File: my-cart/product-list/product.css [hunk 2]
```diff
@@ -0,0 +1,192 @@
...
+}
+.card-text{
+    padding-top: .5rem;
+    padding-bottom: .5rem;
+}
+.add-cart{
+    border: none;
+    padding: .5rem;
+    background-color:var(--SECONDERY-color);
+    color: va;
```
Comment (Adarsh-Patel-Dev) [inline my-cart/product-list/product.css]: What is va here?

File: my-cart/product-list/product.html [hunk 3]
```diff
@@ -0,0 +1,357 @@
...
+            <div class="filter flex-row flex-justify-space-between">
+                <h2>Filter</h2>
+                <button type="reset">Clear</button>
+            </div>
+            <div class="price flex-col">
+                <h2>Price</h2>
+                <!-- range input label -->
+                <div class="price-label flex-row flex-justify-space-between">
+                       <p>
+                        1k</p> <p>
```
Comment (Adarsh-Patel-Dev) [inline my-cart/product-list/product.html]: use proper formatting here

**Review by Adarsh-Patel-Dev — CHANGES_REQUESTED** [review]

**Review by Adarsh-Patel-Dev — APPROVED** [review]

**Pull request merged**